### PR TITLE
Implement asd partial read

### PR DESCRIPTION
**Pull request by domsj**
and use it when no fragment cache is configured

### CODE REVIEW

File: ocaml/src/asd_server.ml [hunk 1]
```diff
@@ -581,6 +581,71 @@ let execute_query : type req res.
...
+                (fun acc (_, length) ->
+                 acc + 200 + length)
+                0
+                slices
+            in
+            perform_read
+              io_sched
+              prio
+              (fun () ->
+               Lwt_log.ign_debug_f "PartialGet for %s" (Slice.show key);
```
Comment (toolslive) [inline ocaml/src/asd_server.ml]: why the `ign_` ?

File: ocaml/src/fragment_cache_test.ml [hunk 2]
```diff
@@ -268,10 +281,55 @@ let test_long () =
...
   in
-  run_with_fragment_cache 20_000L _inner "test_long"
+  run_with_local_fragment_cache 20_000L _inner "test_long"
+
+let _test_lookup2 (cache : cache) =
+  let bid = 0l in
+  let key = "key" in
+  let size = 1_000_000 in
+  let value = Lwt_bytes.create size in
+  let value' = Lwt_bytes.to_string value in
```
Comment (toolslive) [inline ocaml/src/fragment_cache_test.ml]: no equality on bigarray?

File: ocaml/src/asd_server.ml (hunk 1 above)
Comment (domsj) [inline ocaml/src/asd_server.ml]: I copy pasted that from the MultiGet(2) implementation, but it is indeed not needed, so removed the ign_.

File: ocaml/src/fragment_cache_test.ml (hunk 2 above)
Comment (domsj) [inline ocaml/src/fragment_cache_test.ml]: apparently there is, but for some reason I assumed there wasn't. adapting the code.

**Pull request merged**